gno.land/cmd/gnoland/start.go

```diff
@@ -394,19 +391,21 @@ func generateGenesisFile(genesisFile string, pk crypto.PubKey, c *startCfg) erro
 		return fmt.Errorf("unable to load genesis balances file %q: %w", c.genesisBalancesFile, err)
 	}
 
-	// Load examples folder
-	examplesDir := filepath.Join(c.gnoRootDir, "examples")
-	pkgsTxs, err := gnoland.LoadPackagesFromDir(examplesDir, genesisDeployAddress, genesisDeployFee)
-	if err != nil {
-		return fmt.Errorf("unable to load examples folder: %w", err)
-	}
-
 	// Load Genesis TXs
 	genesisTxs, err := gnoland.LoadGenesisTxsFile(c.genesisTxsFile, c.chainID, c.genesisRemote)
 	if err != nil {
 		return fmt.Errorf("unable to load genesis txs file: %w", err)
 	}
 
+	signer := genesisTxs[0].Msgs[0].GetSigners()[0]
```

Comment:
the first tx is not necessarily what has been passed to `TxsAddPackages` command

also loading the examples here will be removed from what I understand so maybe we should not change his file until https://github.com/gnolang/gno/issues/1952 is fixed

Reply:
thanks for the review @n0izn0iz changed back while waiting for that change :)